public/javascripts/webrtc.js

```diff
@@ -336,6 +338,29 @@ David end
 /**
 Michael start
 */
+function handleUsernameForm(event) {
+  event.preventDefault();
+  const form = event.target;
```

Comment:
const form = event.currentTarget;
https://stackoverflow.com/questions/10086427/what-is-the-exact-difference-between-currenttarget-property-and-target-property